spec/requests/api/v1/user_update_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe 'PATCH /api/v1/users/:user_id' do
+  it 'updates a user setting preference' do
+    user = create(:user)
+
+    request_body = {
+                      "music_genre": "piano",
+                      "dyslexic_font": true,
+                      "dark_mode": false,
+                      "font_size": 10
+                    }
```

Comment:
I think Ruby convention might call for this (and the others below) to be written:
```
request_body = {
  "music_genre": "piano",
  "dyslexic_font": true,
  "dark_mode": false,
  "font_size": 10
}
```
I am not 100% certain though, and since its a test, I'll leave it up to decide how to format the hashes here.

Reply:
According to RuboCop... this appears to be OK 
https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_layout.md#enforcedstyle-align_braces

Reply:
Sounds good. I wasn't sure if it was technically a new level, and it actually is nice and readable as is.